ovos_number_parser/util.py

```
@@ -211,3 +258,622 @@ def convert_to_mixed_fraction(number, denominators=range(1, 21)):
+                                                    scale=scale,
+                                                    digits=digits, gender=gender)
+
+            decimal_pronunciation_parts = []
+            #  pronounce decimals either as a whole number or digit by digit
+            if decimal_part_str:
+                if digits == DigitPronunciation.FULL_NUMBER:
+
+                    # handle leading zeros
+                    no_z = decimal_part_str.lstrip("0") # without zeros
+                    n_zeros = len(decimal_part_str) - len(no_z)
+                    for i in range(n_zeros):
```

Ruff (0.14.7)

Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)